### PR TITLE
2.0 history

### PR DESCRIPTION
**Pull request by homm**
Check spelling plz.

### CODE REVIEW

File: HISTORY.markdown [hunk 1]
```diff
@@ -1,5 +1,30 @@
 # History
 
+### 2.0.0, 
+
+*Breaking compatibility changes in this release.*
```
Comment (dmitry-mukhin) [inline HISTORY.markdown]: comp braking

File: HISTORY.markdown [hunk 2]
```diff
@@ -1,5 +1,30 @@
...
+
+#### New features and fixes
+
+* Image resize on client before uploading (`image-shrink` option).
```
Comment (dmitry-mukhin) [inline HISTORY.markdown]: Resize images on client

File: HISTORY.markdown [hunk 3]
```diff
@@ -1,5 +1,30 @@
...
+* Autostore option was removed (but not autostore itself).
```
Comment (dmitry-mukhin) [inline HISTORY.markdown]: , see below)

Comment (dmitry-mukhin) [inline HISTORY.markdown]: а. показалось что строчка ниже это пункт

File: HISTORY.markdown [hunk 4]
```diff
@@ -1,5 +1,30 @@
...
+  Now file is automatically stored if project allows autostore.
+* New option `do-not-store` is added when autostore for your project is enabled
```
Comment (dmitry-mukhin) [inline HISTORY.markdown]: странная фраза

Comment (dmitry-mukhin) [inline HISTORY.markdown]: new option do-not-store. use it when autostore is enabled in project but you don't want to store files uploaded with particular widget(s)

Comment (homm) [inline HISTORY.markdown]: is added for cases when autostore is enabled for your project,
?

File: HISTORY.markdown [hunk 5]
```diff
@@ -1,5 +1,30 @@
...
+  but you don't want to store files on uploading for some reasons.
+* Common widget constructor `AnyWidget` both for single and multiple widgets.
+* Fixed group info `cdnUrl` property when group creation is failed.
+* Assigning of `null` value for multiple widget doesn't break it.
+ 
+#### Breaking changes
+
+* `path-value` is on by default. Widget will always return URL to file,
+  not only when crop is used.
+* Widget value no longer cleared when file is failed to upload.
```
Comment (dmitry-mukhin) [inline HISTORY.markdown]: тут и ниже мб дописать что previous value remains until upload of a new file(s) is successful

**Pull request merged**